testing/dart/window_hooks_integration_test.dart

```diff
@@ -125,65 +137,99 @@ void main() {
     test('onPointerDataPacket preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      PointerDataPacket data;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onPointerDataPacket = (_) {
+        window.onPointerDataPacket = (PointerDataPacket value) {
           runZone = Zone.current;
+          data = value;
         };
       });
 
-      _dispatchPointerDataPacket(new ByteData.view(new Uint8List(0).buffer));
+      final ByteData testData = new ByteData.view(new Uint8List(0).buffer);
+      _dispatchPointerDataPacket(testData);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(data.data, equals(_unpackPointerDataPacket(testData).data));
     });
 
     test('onSemanticsEnabledChanged preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      bool enabled;
 
       runZoned(() {
         innerZone = Zone.current;
         window.onSemanticsEnabledChanged = () {
           runZone = Zone.current;
+          enabled = window.semanticsEnabled;
         };
       });
 
       _updateSemanticsEnabled(window._semanticsEnabled);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(enabled, isNotNull);
+      expect(enabled, equals(window._semanticsEnabled));
     });
 
     test('onSemanticsAction preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      int action;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onSemanticsAction = (_, __) {
+        window.onSemanticsAction = (int value, _) {
           runZone = Zone.current;
+          action = value;
         };
       });
 
-      _dispatchSemanticsAction(0, 0);
+      _dispatchSemanticsAction(1234, 0);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(action, equals(1234));
     });
 
     test('onPlatformMessage preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      String name;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onPlatformMessage = (_, __, ___) {
+        window.onPlatformMessage = (String value, _, __) {
           runZone = Zone.current;
+          name = value;
         };
       });
 
-      _dispatchPlatformMessage(null, null, null);
+      _dispatchPlatformMessage("testName", null, null);
```

Comment:
nit: use single quotes unless there's a reason to use double quotes

Reply:
Done.  (It's my years of C++ fingers that need retraining...)